### ISSUE TITLE
KeyError at init

### ISSUE THREAD

**Issue by luzik** [thomasloven/hass-browser_mod#415]
After updating to latest version got this in logs

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
    await func(hass, connection, msg)
  File "/config/custom_components/browser_mod/connection.py", line 143, in handle_update
    dev = getBrowser(hass, browserID)
  File "/config/custom_components/browser_mod/browser.py", line 197, in getBrowser
    browsers[browserID] = BrowserModBrowser(hass, browserID)
  File "/config/custom_components/browser_mod/browser.py", line 42, in __init__
    self.update_entities(hass)
  File "/config/custom_components/browser_mod/browser.py", line 71, in update_entities
    _assert_browser_sensor("sensor", "path", "Browser path", icon="mdi:web")
  File "/config/custom_components/browser_mod/browser.py", line 65, in _assert_browser_sensor
    adder = hass.data[DOMAIN][DATA_ADDERS][type]
KeyError: 'sensor'
```

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [x ] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [x ] Have made sure I am using the latest version of the plugin.

- [x ] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [ x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by LeoCal** [comment]
I got the exact same issue in browser_mod after updating to HA 2022.9.

I'm now suspecting that the issue I'm hitting with 2022.9 where the entire home assistant instance becomes unresponsive and stuck might be due to this - but it's just speculation for now, I've not tried disabling browser_mod to see if this is really the case.

For now, I've just switched back to 2022.8.7 where everything is working smoothly for me, but at some point I would need to nail down the issue in my setup.
